src/components/DepthChart/Chart/Chart.tsx

```diff
@@ -213,14 +223,15 @@ class Chart extends React.Component<ChartProps> {
     this.graphics = [];
     this.width = this.props.width;
     this.height = this.props.height;
-    this.midXAsks = this.width / 2 + Math.round(chart.strokeWidth / 2);
-    this.midXBids = this.width / 2 - Math.round(chart.strokeWidth / 2);
+    this.midXAsks = this.props.width / 2 + Math.round(chart.strokeWidth / 2);
+    this.midXBids = this.props.width / 2 - Math.round(chart.strokeWidth / 2);
     this.midY = this.height;
-    this.asks = this.props.asks.reverse();
+    this.asks = reverse(this.props.asks);
```

Comment:
is it necessary to define these fields? you can get it directly from the props in your methods. If you need to reverse asks you can do it in the index.tsx